docs/github.md

```diff
@@ -0,0 +1,245 @@
+to require approval for new Personal Access Token access.
+
+As in the
+[Allow Usage of Personal Access Tokens](#allow-usage-of-personal-access-tokens)
+section, you first need to navigate to
+*Organization Settings > Third-party Access section > Personal access tokens*
+
+![Organization's personal access token approvals](images/github/organizationSettings.jpg)
+
+From there, open "Pending requests" and review the token request. If the requested
+access is in line with the description from 
+[Create a Personal Access Token](#create-a-personal-access-token), then approve the
+token request.
+
+### Requiring Sematic Checks to Pass
+
+This section describes how you can configure your GitHub to require Sematic checks
+to pass before allowing commits to merge to your protected branches (e.g.
+`main`/`master`). **It should only be performed once you have successfully**
+**executed a Sematic check on one of your commits in your repository.**
+If you attempt it before that point, you will not see Sematic's check as an
+option to require, and you may risk blocking PRs before you're ready to
+have Sematic checks performed.
+
+Once you're ready, go to the repository you wish to add this protection to,
+and navigate to its settings page. From there, select "Branches" from the
+side menu.
+
+![Configuring branch protection](images/github/branchProtection.jpg)
+
+Either edit an existing branch protection rule or add a new one (with the
+"Add rule" button), depending on whether the branch you're protecting already
+has other protections in place.
+
+Once you're editing the branch protection, ensure the branch name pattern matches
+the branch you wish to protect, then go to the "Protect matching branches"
+configuration box. There, enable "Require status checks to pass before merging".
+
+You may then search for the `sematic-pipelines-pass` check, and add it to the
+list of required checks.
+
+![Require check](images/github/requireCheck.jpg)
+
+Finally, click the "Save changes" button to finalize your configuration.
+
+## CI Configuration
+
+```TODO: Write```
+This section describes how to configure your CI to leverage Sematic's GitHub
+integration. Since everyone's CI and infrastructure setup is different, the
+details of this step will differ for everyone. However, the high-level steps
+remain the same:
+
+1. Ensure your CI has network access to your Sematic deployment
+2. Get a Sematic API key to use via your CI. This can just be the API key of
+an individual user.
+3. Set the `SEMATIC_API_ADDRESS` environment variable on your CI worker to
+point to your Sematic deployment.
+4. Set the `SEMATIC_API_KEY` environment variable on your CI worker to
+enable Sematic access.
+5. Configure your CI to run the command line command to launch your Sematic
+pipelines. Any run that should be considered as a requirement for the check
+to pass should have the tag `checks-commit:<git sha being built by CI>`.
```

Comment:
Please add a `Tag` entry to the Glossary, then link that here.

Reply:
done